apps/web-giddh/webpack.partial.js

```diff
@@ -1,5 +1,6 @@
 const webpack = require('webpack');
 const CompressionPlugin = require('compression-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
Avoid using `webpack-bundle-analyzer` as `source-map-explorer` is recommended by Angular team